### DISCUSSION TITLE
Installing OpenRadioss on a cluster

### DISCUSSION THREAD

**Discussion by MariaLauraSantoni** [OpenRadioss/OpenRadioss#2803] [Q&A]
Hi everyone,

I am trying to install OpenRadioss from source inside a cluster but I am encountering a lot of problems because I do not have permissions and the main problem is the installation of gfortran. I tried with a conda environment but it is not working.

Have you ever tried to install OpenRadioss in a cluster? If so, I would really appreciate some hints.

Thanks,
Maria Laura

**Comment by MarianBulla** [comment]
Hi @MariaLauraSantoni,

thanks for your interest on OpenRadioss and a very warm welcome in this community.
Regarding the installation of OpenRadioss on a Linux system, did you watch the video on youtube: 

https://www.youtube.com/watch?v=ddvH2CNfaYg&t=601s

It should work, as long as you can install all necessary software in your home directory.
For global installation, you have to ask you sys-admins.

Hope it helps you ?

Thanks and best regards,
Marian
